ledger/tracker.go

```diff
@@ -499,24 +531,26 @@ func (tr *trackerRegistry) commitRound(dcc *deferredCommitContext) {
 		lt.postCommitUnlocked(tr.ctx, dcc)
 	}
 
+	return nil
 }
 
 // initializeTrackerCaches fills up the accountUpdates cache with the most recent ~320 blocks ( on normal execution ).
 // the method also support balances recovery in cases where the difference between the lastBalancesRound and the lastestBlockRound
 // is far greater than 320; in these cases, it would flush to disk periodically in order to avoid high memory consumption.
-func (tr *trackerRegistry) initializeTrackerCaches(l ledgerForTracker) (err error) {
+func (tr *trackerRegistry) initializeTrackerCaches(l ledgerForTracker, cfg config.Local) (err error) {
```

Comment:
The only place I saw this called, the supplied cfg was tr.cfg. So maybe we can drop the extra argument since tr contains the cfg now.

Reply:
fixed